portal/src/main/webapp/js/src/civic/civicservice.js

```diff
@@ -0,0 +1,164 @@
+
+function CivicService() {
+    
+    // All results are stored here
+    var civicGenes = {};
+    
+    function getCivicGenesBatch(ids) {
+        return $.ajax({
+            type: 'GET',
+            url: civicUrl + 'genes/' + ids,
+            dataType: 'json',
+            data: {
+                identifier_type: 'entrez_symbol'
+            }
+        }).then(function(result) {
+            if (_.isString(result)) {
+                result = $.parseJSON(result);
+            }
+
+            // Wrap the result in an Array if it is not already one
+            if (!(result instanceof Array)) {
+                result = [result];
+            }
+            if (result) {
+                result.forEach(function(record) {
+                    var civicGene = {
+                        id: record.id,
+                        name: record.name,
+                        description: record.description,
+                        url: 'https://civic.genome.wustl.edu/#/events/genes/'
```

Comment:
This is the link to the website entry, which is (at least now) not configurable, as opposed to the API url.